content/en/dora_metrics/setup/deployments.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Sending Deployment Events for DORA Metrics
+kind: documentation
+description: Learn how to send deployment events for DORA Metrics.
+aliases:
+- /continuous_integration/dora_metrics/setup/deployments
+is_beta: true
+further_reading:
+- link: "https://app.datadoghq.com/release-notes?category=Software%20Delivery"
+  tag: "Release Notes"
+  text: "Check out the latest Software Delivery releases! (App login required)"
+- link: "/tracing/service_catalog"
+  tag: "Documentation"
+  text: "Learn about the Service Catalog"
+- link: "https://github.com/DataDog/datadog-ci"
+  tag: "Source Code"
+  text: "Learn about the datadog-ci CLI tool"
+- link: "/continuous_delivery/deployments"
+  tag: "Documentation"
+  text: "Learn about Deployment Visibility"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">DORA Metrics is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+{{< callout url="https://forms.gle/Eqq6uXfGjYxmqpjDA" header="false" >}}
+DORA Metrics are in private beta. Fill out the form below to be added to the waitlist.
```

Comment:
Quick copy update:
"DORA Metrics private beta is closed. Fill out the form below to be added to the waitlist."